R/anovarepeatedmeasures.R

```diff
@@ -1478,9 +1484,17 @@ AnovaRepeatedMeasures <- function(jaspResults, dataset = NULL, options) {
 
   postHocTable$addColumnInfo(name="t.ratio", title=gettext("t"), type="number")
 
-  if (options$postHocTestEffectSize && !interactionTerm) {
+  if (options$postHocTestEffectSize) {
     postHocTable$addColumnInfo(name="cohenD", title=gettext("Cohen's d"), type="number")
-    postHocTable$addFootnote(gettext("Cohen's d does not correct for multiple comparisons."))
+    
+    if (!is.null(options$postHocTestPooledError) && !options$postHocTestPooledError)
+      postHocTable$addFootnote(gettext("Computation of Cohen's d based on pooled error."))
```

Comment:
Emmeans only works with pooled errors (the post hoc test that does not assume pooled errors is something I wrote myself, and also does not work for interactions), so that's why here I include that message, because it is not something the user would expect. Is there a way to show a warning instead of a note (or something that looks more eyecatching?)

Reply:
Ah, ok, that makes sense now!

There is no way to show info or warning boxes, but it's on a to-do list. So for now this is ok